### PR TITLE
Enable log collection for chaos test runs

### PR DESCRIPTION
**Pull request by chaitanyaenr**

### CODE REVIEW

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

File: ci-operator/step-registry/redhat-chaos/container-scenarios/etcd-hangup/redhat-chaos-container-scenarios-etcd-hangup-ref.yaml [hunk 1]
```diff
@@ -50,6 +50,10 @@ ref:
       default: "0"
     - name: TELEMETRY_ARCHIVE_SIZE
       default: "1000000"
+    - name: TELEMETRY_LOGS_BACKUP
+      default: "True"
+    - name: TELEMTRY_CLI_PATH
+      default: "/usr/local/bin/oc"
```
Comment (paigerube14) [inline ci-operator/step-registry/redhat-chaos/container-scenarios/etcd-hangup/redhat-chaos-container-scenarios-etcd-hangup-ref.yaml:56]: Should the default oc cli be in the path already?

Comment (chaitanyaenr) [reply ci-operator/step-registry/redhat-chaos/container-scenarios/etcd-hangup/redhat-chaos-container-scenarios-etcd-hangup-ref.yaml:56]: We copy the binary to /usr/local/bin - https://github.com/redhat-chaos/krkn/blob/main/containers/Dockerfile#L25, adding the path explicitly to make sure it finds it in cases where /usr/local/bin is not part of the PATH. Thoughts?

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Pull request merged**